bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/CheckAndMutateUtil.java

```diff
@@ -80,13 +79,13 @@ public static boolean wasMutationApplied(
 
   /**
    * This class can be used to convert HBase checkAnd* operations to Bigtable
-   * {@link CheckAndMutateRowRequest}s.
+   * {@link ConditionalRowMutation}s.
    */
   public static class RequestBuilder {
     private final HBaseRequestAdapter hbaseAdapter;
-    private final CheckAndMutateRowRequest.Builder requestBuilder;
 
-    private final List<Mutation> mutations = new ArrayList<>();
+    private final com.google.cloud.bigtable.data.v2.models.Mutation mutations =
```

Comment:
Is there a reason we can't import `com.google.cloud.bigtable.data.v2.models.Mutation`? That would make the code look nicer.